biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderParser.java

```diff
@@ -37,15 +37,57 @@
 
 public class GenericGenbankHeaderParser<S extends AbstractSequence<C>, C extends Compound> implements SequenceHeaderParserInterface<S,C> {
 
+	@SuppressWarnings("unused")
+
+	// Brief description of sequence; includes information such as source organism,
+	// gene name/protein name, or some description of the sequence's function (if
+	// the sequence is non-coding). If the sequence has a coding region (CDS),
+	// description may be followed by a completeness qualifier, such as "complete
+	// cds".
+	private String description;
```

Comment:
Nice docs! Could you make them into javadocs? i.e. surround comments by `/**   */` . The same for others below

Reply:
Sure this is now done